csrc/parallel_dimension_map.cpp

```diff
@@ -196,32 +87,26 @@ void ParallelDimensionMap::adjustMappingsForWarpPadding() {
   }
 
   const auto tidx_pt = ParallelType::TIDx;
-  auto warp_size = at::cuda::warp_size();
```

Comment:
`at::cuda::warp_size();` was added to support ROCm, which we no longer care anymore.